modules/core/internal/telemetry/client.go

```diff
@@ -0,0 +1,58 @@
+package telemetry
+
+import (
+	metrics "github.com/hashicorp/go-metrics"
+
+	"github.com/cosmos/cosmos-sdk/telemetry"
+
+	ibcmetrics "github.com/cosmos/ibc-go/v8/modules/core/metrics"
+)
+
+func ReportCreateClient(clientType string) {
```

Comment:
Should we add doc strings to all these public functions?

Reply:
That might be nice. In case someone is wondering if they should use them or not, a bit of explanation on how and when they are supposed to be used.

Potentially a small text or readme even on how we use telemetry?